preprocessing/edit_labels.ipynb

```diff
@@ -0,0 +1,142 @@
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Remove neutral reviews\n",
+    "test_df_no_neutral = test_df[test_df['star_rating'] != \"neutral\"]\n",
+    "test_df_no_neutral.head(10)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No-neutrals Dataframe shape\n",
+      "Rows:  520000 \n",
+      "Columns:  3\n"
+     ]
+    }
+   ],
+   "source": [
+    "rows, columns = test_df_no_neutral.shape\n",
+    "print(\"No-neutrals Dataframe shape\\nRows: \", rows, \"\\nColumns: \", columns)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "outputs": [],
+   "source": [
+    "# To file\n",
+    "# test_df_no_neutral.to_csv(path_or_buf = \"no_neutrals.csv\", index = False)"
```

Comment:
We need to make sure all our `.csv` files are in a consistent format: all fields should be wrapped in double quotes (`"`), with double quote escaping. If you try using a `csv_reader` to open the file output by this line, it fails! Try specifying the `.csv` file specifications using the parameters listed on lines 14-18 of `dataset.py` and the [documentation](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html) for `pandas`'s `DataFrame.to_csv` method.

Reply:
Or, you could just use `dataset.csv_writer` to accomplish this. You don't have to if you really don't want to... but I did spend a not insignificant amount of time working on it and would be sad if it never got used. I actually implemented the remapping as an example on lines 53-71 of `test_install.py`

Reply:
Why do we want quotation marks around our fields? We can just type them when we import them + no particular format has been defined, since we don't use them as input anywhere. With regards to what code we'll use, we can talk about it on Monday - I'm partial to dataframes, since they're establised and there's tons of documentation online🤠

If we want to encapsulate the fields, I added the line for it, but it's currently commented out👌

Reply:
The csv file format is kind of a shitshow, it was never standardized before it became commonly used and there are now multiple differing file format standards which all use the same csv extension. The encapsulating quotation marks in the output csv are used to delimit the individual fields (label, title, body) and are independent of the Python type we are importing to; it seems much easier to me to just use the same csv formatting parameters for all our data files and not have to deal with weird edge cases when importing different csvs which aren't formatted identically, so I would ask that we do please use standardized file formats.

And I agree on talking over our code architecture on Monday, I have performance concerns about using dataframes when we move to GPU computing but I can discuss those with you later.